planex/Makefile.rules

```diff
@@ -34,6 +34,10 @@ DEPEND ?= planex-depend
 DEPEND_FLAGS ?= -d $(DIST) --topdir $(TOPDIR) --ignore-from ignore \
                 --pins-dir $(PINSDIR) $(DEPEND_EXTRA_FLAGS)
 
+ifndef VERBOSE
```

Comment:
Could you please make this depend on the QUIET flag instead of adding a new flag?